vendor/bat-native-ads/src/bat/ads/internal/processors/contextual/text_embedding/text_embedding_processor_unittest.cc

```diff
@@ -0,0 +1,56 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_processor_util.h"
+
+#include <map>
+
+#include "bat/ads/internal/base/unittest/unittest_base.h"
+#include "bat/ads/internal/base/unittest/unittest_file_util.h"
+
+// npm run test -- brave_unit_tests --filter=BatAds*
+
+namespace ads {
+namespace processor {
+
+class BatAdsTextEmbeddingProcessorTest : public UnitTestBase {
+ protected:
+  BatAdsTextEmbeddingProcessorTest() = default;
+
+  ~BatAdsTextEmbeddingProcessorTest() override = default;
+};
+
+TEST_F(BatAdsTextEmbeddingProcessorTest, SanitizeTextHtml) {
```

Comment:
This test only tests Html; could we please break out and test both HTML and text? See other comment about Text/HTml